src/libraries/System.Private.CoreLib/src/System.Private.CoreLib.Shared.projitems

```diff
@@ -393,6 +393,7 @@
     <Compile Include="$(MSBuildThisFileDirectory)System\IO\BinaryReader.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)System\IO\BinaryWriter.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)System\IO\BufferedStream.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)System\IO\BufferedFileStreamStrategy.cs" />
```

Comment:
Can we consider creating a sub-folder for _all_ `*FileStream*.cs` files?

Reply:
I think if we want these files in a subfolder, we would have to be consistent and also put them inside their own subnamespace. Something like `System.IO.FileStreamStrategies`.

But as I said in my comment, I think we should not block on this. We can do this in a future PR, if everyone agrees.